src/pages/dashboard/InstitutionFiltering.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { Dispatch, FC, SetStateAction, useEffect, useState } from 'react';
+import { Checkbox, FormControl, FormControlLabel, FormGroup } from '@material-ui/core';
+import FormLabel from '@material-ui/core/FormLabel';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { AllDLRInstitutionNames, QueryObject } from '../../types/search.types';
+
+const StyledFormControl: any = styled(FormControl)`
```

Comment:
```suggestion
const StyledFormControl = styled(FormControl)`
```

Reply:
For å kunne sette component="fieldset" . mulig UU-greier

Reply:
fieldset er UU-greier ja